utils/helpers/mergeDeep.cjs

```diff
@@ -0,0 +1,25 @@
+const isObject = (variable) => {
+  return typeof variable === 'object' && !Array.isArray(variable) && variable !== null;
+};
+
+const mergeDeep = (baseObj, incomingObj) =>
```

Comment:
Meestal zie je bij `merge`/`assign` functies dat het eerste object wordt aangepast (er ge-`return`ed). Dat gebeurt hier niet. Ik weet niet zo goed wat ik er van vind: of dat wel of juist niet moet gebeuren. Wat vind jij?

Reply:
Deze keuze zou perfect passen in de jsdoc die ik elders in deze PR noemde.

Reply:
Aangezien het een losse functie, vind ik het duidelijker om het niet aan te passen. Ook meer in lijn van Functioneel programmeren en geen mutable values te hebben.

Reply:
Fair enough